youtube_dl/extractor/extractors.py

```diff
@@ -731,8 +731,9 @@
     NJoyEmbedIE,
 )
 from .ndtv import NDTVIE
-from .netzkino import NetzkinoIE
+from .nebula import NebulaIE
```

Comment:
Add further extractors from back-port:
```suggestion
from .nebula import (
    NebulaIE,
    NebulaChannelIE,
    NebulaClassIE,
    NebulaSubscriptionsIE,
)
```